### PR TITLE
Updated with review comments

### PR DESCRIPTION
**Pull request by WilliamDenniss**

### CODE REVIEW

File: draft-ietf-oauth-device-flow.xml [hunk 1]
```diff
@@ -234,9 +237,25 @@
 
       <section anchor="AuthorizationRequest" title="Device Authorization Request">
 
+  <t>This specification defines a new OAuth endpoint, the device
+  authorization endpoint. This is separate from the OAuth authorization
+  endpoint with which the user interacts with via a user-agent (i.e. a browser). In the
```
Comment (selfissued) [inline draft-ietf-oauth-device-flow.xml]: i.e. is always followed by a comma.

File: draft-ietf-oauth-device-flow.xml [hunk 2]
```diff
@@ -311,7 +330,7 @@
 	    <t hangText="interval"> <vspace/>
 	    OPTIONAL.  The minimum amount of time in seconds that the
 	    client SHOULD wait between polling requests to the token
-	    endpoint.</t> 
+	    endpoint. If no value provided, clients should use 5 as the default.</t>
```
Comment (selfissued) [inline draft-ietf-oauth-device-flow.xml]: Add "is" after "value".
Change "should" to "SHOULD".

**Review by selfissued — APPROVED** [review]
Looks good.  A few nits are identified in comments on individual lines.

File: draft-ietf-oauth-device-flow.xml [hunk 3]
```diff
@@ -239,8 +239,9 @@
 
   <t>This specification defines a new OAuth endpoint, the device
   authorization endpoint. This is separate from the OAuth authorization
-  endpoint with which the user interacts with via a user-agent (i.e. a browser). In the
-  case of the device authorization endpoint, the client on the device
+  endpoint defined in <xref target="RFC6749"/> with which the user interacts
+  with via a user-agent (i.e. a browser). With the device authorization
```
Comment (selfissued) [inline draft-ietf-oauth-device-flow.xml]: i.e. must be followed by a comma

**Review by selfissued — APPROVED** [review]
See the nit in the individual comment

File: draft-ietf-oauth-device-flow.xml (hunk 1 above)
Comment (WilliamDenniss) [reply draft-ietf-oauth-device-flow.xml]: DONE

File: draft-ietf-oauth-device-flow.xml (hunk 2 above)
Comment (WilliamDenniss) [reply draft-ietf-oauth-device-flow.xml]: DONE. And I think it should be a MUST?

File: draft-ietf-oauth-device-flow.xml (hunk 3 above)
Comment (WilliamDenniss) [reply draft-ietf-oauth-device-flow.xml]: DONE

**Review by selfissued — APPROVED** [review]

**Review by selfissued — APPROVED** [review]

**Pull request merged**